### PR TITLE
Add drone-plugins/drone-gpgsign plugin

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

File: content/drone-plugins/drone-anynines/index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
```
Comment (techknowlogick) [inline content/drone-plugins/drone-anynines/index.md]: Should this file be in a different PR?

Comment (tboerger) [reply content/drone-plugins/drone-anynines/index.md]: Good catch, I used the wrong base branch :D
